packages/terra-clinical-item-display/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Add `strong` & `inactive` text styles.
```

Comment:
This needs to be updated. It should listed as `Added` since they didn't previously exist and we want to indicate `isInvalid` is a new prop.